portfolio/banking/banking.go

```diff
@@ -0,0 +1,95 @@
+package banking
```

Comment:
i know you will hate me for suggesting this but what are your thoughts on changing banking -> bank

Reply:
I intially liked `banking` because it would have a distinction from a `bank` struct or var. But the type is called `Account`, so I'm happy with `bank` being the package name too

Reply:
called it banking orginally because it dealt with multiple banking related things but happy for it to be whatever people fit best
@thrasher- @gloriousCode @shazbert @MadCozBadd

Reply:
I don't mind